src/components/MediaServiceProviderBox.tsx

```diff
@@ -0,0 +1,24 @@
+import styled from "styled-components";
+
+import { cssScale, colors, dropShadow } from "src/common/constants";
+
+interface IMediaServiceProviderBox{
+  width?: string,
+  height?: string
+}
+
+const defaultWidth = 190;
+const defaultHeight = 115;
```

Comment:
These should be pulled out to the route level component, please check for other occurrences.

Reply:
`route level component` I did not get this.
These values must be pulled out to the page using this(similar) component and then passing it further as props to component?

Reply:
The `SelectVideo` component